cfme/tests/middleware/test_middleware_provider.py

```diff
@@ -109,7 +109,8 @@ def test_required_fields_validation(provider, soft_assert):
 @pytest.mark.usefixtures('setup_provider')
 def test_duplicite_provider_creation(provider):
     """Tests that creation of already existing provider fails."""
-    with error.expected('Host Name has already been taken'):
+    with error.expected('Name has already been taken, '
```

Comment:
REQUIRED: tests fails in 5.7 with flash message 
"Name has already been taken, Host Name has already been taken"